sk.json

```diff
@@ -141,9 +149,81 @@
             "title" : "Randomizér",
             "line1" : "Zvoľ si náhodnú výstroj z hociktorej dostupnej položky.",
             "chooseText" : "Klikne pre zvolenie",
-            "randomise-all" : "Náhodne Všetko"
+            "randomise-all" : "Náhodne Všetko",
```

Comment:
The comma at the end of the line is wrong. You must add `},` right after this line to end the block. The file will break without it because the JSON is invalid.

Reply:
Not the right place to ask, but how do you review changes like this. I mean how do you select the changes specifically? I just took screenshots and sent it on Discord. @Krymonota